saml.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |s|
 
   s.add_dependency "activesupport", ">= 3.2.15"
   s.add_dependency "activemodel", ">= 3.0.0"
-  s.add_dependency "xmlmapper", '~> 0.7.0'
+  s.add_dependency "xmlmapper", '~> 0.7.2'
   s.add_dependency 'nokogiri', '~> 1.6', '>= 1.6.8'
-  s.add_dependency "xmldsig", '~> 0.4.0'
+  s.add_dependency "xmldsig", '~> 0.5.2'

```

Comment:
Oh, thanks for fixing the errors! https://github.com/digidentity/libsaml/commit/ce188fd34b7c531927c516fcc9172f33144010ab